.agents/AGENT-INSTRUCTIONS.md

```diff
@@ -130,6 +132,120 @@ Before starting work, complete these steps IN ORDER:
 
 ---
 
+## Impact Ripple Analysis (Build Configuration Changes)
```

Comment:
@coderabbitai there are many types of impact analysis. when performing changes, agents should be consulted during prior to implementation when using the `planner` agent: `implementer` for code, `architect` for conceptual and domain consistency, `qa` for quality, `security` for security, and `devops` for CI/CD, local developer experiences and operations. These are missing capabilities that should be in the upstream rjmurillo/ai-agents. Generate a PRD as an upstream issue.